### DISCUSSION TITLE
QR Codes causing random recording?

### DISCUSSION THREAD

**Discussion by spencer245** [gopro/labs#1116] [Q&A]
Hello all,

I am currently setting up 4 GoPro Hero 12's for 6 month long timelapse covering the construction of a building. I have taken some time to learn the gopro labs command language a decent bit and came up with this line of code to be converted into a QR code:

!T:ON07:00OFF17:00I:120P:1WDW12345;

The goal of this line of code is to turn the GoPros on at 7:00 AM, take a photo every 2 minutes, have the cameras turn on/off between photos for battery preservation, until 5 PM every week day (M-F). 

All seems normal up to this point. I input this string of code, receive a new QR code, and scan it with my GoPro Hero 12's. However, upon scanning this qr code, the GoPro immediately starts recording in "Real Speed" or whatever its called. I don't understand why this is happening and I can't seem to get it out of this mode, despite explicitly using the "P" for photo mode. 

I'd love some help here. 

Extra question if anyone feels inclined to answer: do I need to specify all of my settings for code like this? Do i need to set my min and max ISO? do i need to set the quality to 4k? or will the gopro just use auto-iso and auto-WB and etc?

Thank you to anyone who takes some time to read and/or comment on this post!

**Comment by dnewman-gpsw** [comment]
That is not a valid command. Documentation is here https://gopro.github.io/labs/control/tech/

There is no "ON" and "OFF" command. !T is for Timewarp control.  No where in that command do you request a Photo mode.

; <- are never used.  Who advised you on this?  There is near zero related to Labs commands here.  Looks like you used ChatGPT for this.
